infra-as-code/bicep/CRML/subscriptionAlias/subscriptionAlias.bicep

```diff
@@ -13,6 +15,15 @@ param parSubscriptionName string
 @description('The full resource ID of billing scope associated to the EA, MCA or MPA account you wish to create the subscription in.')
 param parSubscriptionBillingScope string
 
+@description('Tags you would like to be applied')
+param parTags object = {}
+
+@description('The ID of the management group where the subscription will be placed. Also known as the parent management group.')
+param parManagementGroupId string
+
+@description('The ID of the responsible user for the subscription.')
```

Comment:
this can be a group or anything right. Just an object ID?

Shall we updated the decorator to suggest this, not just a user?